### PR TITLE
Add liveness endpoint

### LINKED ISSUE #3283: Add /liveness endpoint
### Description
Add a liveness endpoint that returns 200 OK all the time.  This allows checking that the node is up even if it's syncing etc.

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Add liveness endpoint

## Fixed Issue(s)

Fixes #3283 

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+
+public class Liveness implements Handler {
+
+  public static final String ROUTE = "/teku/v1/admin/liveness";
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java:30]: need to add this to the `RestApiV1Test` as an endpoint

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      path = ROUTE,
+      method = HttpMethod.GET,
+      summary = "Returns 200 if the node is up even if it is syncing.",
+      tags = {TAG_TEKU},
+      responses = {
+        @OpenApiResponse(status = RES_OK),
+      })
+  @Override
+  public void handle(final Context ctx) throws Exception {
+    ctx.status(SC_OK);
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java:43]: `ctx.header(Header.CACHE_CONTROL, CACHE_NONE);`

File: data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/LivenessTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  public void setup() {
+    handler = new Liveness();
+  }
+
+  private Context context = mock(Context.class);
+
+  @Test
+  public void shouldReturnOkWhenTekuIsUp() throws Exception {
+    handler.handle(context);
+    verify(context).status(SC_OK);
```
Comment (rolfyone) [inline data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/LivenessTest.java:41]: once the header is in, we should verify it got set here, like the SC_OK.

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java (hunk 1 above)
Comment (cemozerr) [reply data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java:30]: done.

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java (hunk 2 above)
Comment (cemozerr) [reply data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/Liveness.java:43]: done.

File: data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/LivenessTest.java (hunk 3 above)
Comment (cemozerr) [reply data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/admin/LivenessTest.java:41]: done.

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**